bazel/repositories.bzl

```diff
@@ -17,9 +17,9 @@ load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
 def proxy_wasm_cpp_host_repositories():
     http_archive(
         name = "proxy_wasm_cpp_sdk",
-        sha256 = "489768fb95ede507543ee5982610b541a2c5b57216695a9e5c2eb8c83c9d20a3",
-        strip_prefix = "proxy-wasm-cpp-sdk-9af5ac0145a8790f62ca501c43f6fa1ea24d2d93",
-        urls = ["https://github.com/proxy-wasm/proxy-wasm-cpp-sdk/archive/9af5ac0145a8790f62ca501c43f6fa1ea24d2d93.tar.gz"],
+        sha256 = "fb62d0db7a5e94b2bf0bfab72bb967b1d4549d75be81a457dad48d4a22430947",
+        strip_prefix = "proxy-wasm-cpp-sdk-7bf94c86800773ee762be09fb84dcc2972da0b37",
+        urls = ["https://github.com/proxy-wasm/proxy-wasm-cpp-sdk/archive/7bf94c86800773ee762be09fb84dcc2972da0b37.tar.gz"],
```

Comment:
Update after https://github.com/proxy-wasm/proxy-wasm-cpp-sdk/pull/122 is merged.